boards/infineon/cy8ckit_062s2_ai/board.yml

```
@@ -0,0 +1,9 @@
+# Copyright (c) 2024 Arrow Electronics.
+# SPDX-License-Identifier: Apache-2.0
+
+board:
+  name: cy8ckit_062s2_ai
+  full_name: PSOC6 AI Evaluation Kit
```

Pointing this one out as it's important that trademarks are properly used/spelled, especially as this appears in the board listing and can be used to search for boards by name

```suggestion
  full_name: PSOC 6 AI Evaluation Kit
```